pom.xml

```diff
@@ -17,6 +17,8 @@
     <grpc.version>1.25.0</grpc.version>
     <protobuf.version>3.11.0</protobuf.version>
     <protoc.version>3.10.0</protoc.version>
+    <dapr.proto.url>https://raw.githubusercontent.com/dapr/dapr/e540a7166aeaf115773ccc4c7a1056ae7eed073b/pkg/proto/dapr/dapr.proto</dapr.proto.url>
```

Comment:
JFYI - This url is snapshot, which means it will not be a up-to-date file

Reply:
This is by design. We want to have a fixed version. So this build has predictable behavior.

Reply:
How about the version tag instead of hash? e.g. https://raw.githubusercontent.com/dapr/cli/v0.3.0/cmd/dapr.go

Reply:
Yes, that is preferred but this ref is for the 0.4 milestone and we did not have one at that time.

Reply:
Updated to use v0.4.0